CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# CONTRIBUTING
+
+We use GitHub to host code, to track issues and feature requests, as well as accept pull requests.
+
+
+## Issues
+As usual, [check if issue already exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments) in GitHub issue tracker.
+
+## Commit messages
+Please conform to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
+
+
+# Used tools
+
+## Linters
+
+### MegaLinter
+[MegaLinter](https://megalinter.io/latest/) is used in [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow), which checks all the source code.
+
+Can be run locally, but it is pain. See [MegaLinter Runner](https://megalinter.io/latest/mega-linter-runner/).
```

Comment:
I can add a PR to reduce the pain of running this manually later, should be rather straight forward.

Reply:
Sure, sounds good :)